nautobot_floor_plan/api/nested_serializers.py

```diff
@@ -15,16 +15,4 @@ class Meta:
         """Meta attributes."""
 
         model = models.FloorPlan
-        fields = ["id", "url", "x_size", "y_size"]
-
-
-class NestedFloorPlanTileSerializer(WritableNestedSerializer):
-    """FloorPlanTile Nested Serializer."""
-
-    url = serializers.HyperlinkedIdentityField(view_name="plugins-api:nautobot_floor_plan-api:floorplantile-detail")
-
-    class Meta:
-        """Meta attributes."""
-
-        model = models.FloorPlanTile
-        fields = ["id", "url", "x_origin", "y_origin", "x_size", "y_size"]
+        fields = "__all__"
```

Comment:
revert, same question?

Reply:
Drift manager freshly rebakes the cookie from json file and compares changes to the last rebake. In the following rebakes, only changes to the template since the last rebake will be propagated, the PR should be much smaller then. Just this first `setup-cookie` contains most files, as the last rebake is empty and delta means all files in the cookie.